com.meetfranz.Franz.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.meetfranz.Franz.desktop</id>
```

Comment:
```suggestion
  <id>com.meetfranz.Franz</id>
```

Reply:
You will also have to add 
`<launchable type="desktop-id">com.meetfranz.Franz.desktop</launchable>`